Add new troubleshooting failed SMS concept doc (#1162)
Nexmo/nexmo-developer: _documentation/messaging/sms/guides/troubleshooting-sms.md

| 6 | Invalid Message | The platform was unable to process this message, for example, an un-recognized number prefix
| 7 | Number Barred | The number you are trying to submit to is blacklisted and may not receive messages
| 8 | Partner Account Barred  | Your Nexmo account has been suspended. Contact <support@nexmo.com>. |
| 9 | Partner Quota Violation  | You do not have sufficient credit to send the message. Top-up and retry.  |
| 10 | Too Many Existing Binds | The number of simultaneous connections to the platform exceeds your account allocation
| 11 | Account Not Enabled For HTTP | This account is not provisioned for the SMS API, you should use SMPP instead
| 12 | Message Too Long | The message length exceeds the maximum allowed
| 14 | Invalid Signature | The signature supplied could not be verified
| 15 | Invalid Sender Address | You are using a non-authorized sender ID in the `from` field. This is most commonly in North America, where a Nexmo long virtual number or short code is required.  |
| 22 | Invalid Network Code | The network code supplied was either not recognized, or does not match the country of the destination address.

> | 23 | Invalid Callback Url | The callback url supplied was either too long or contained illegal characters

Author: ```suggestion
| 23 | Invalid Callback Url | The callback URL supplied was either too long or contained illegal characters
```

| 29 | Non-Whitelisted Destination | Your Nexmo account is still in demo mode. While in demo mode you must add target numbers to your whitelisted destination list. Top-up your account to remove this limitation. |
| 32 | Signature And API Secret Disallowed | A signed request may not also present an `api_secret`.

> | 33 | Number De-activated | The number you are trying to submit to is de-activated and may not receive messages

Author: ```suggestion
| 33 | Number De-activated | The number you are trying to send messages to has been deactivated and may not receive them
```
